__tests__/itineraries/bike-only.json

```diff
@@ -0,0 +1,491 @@
+{
```

Comment:
I don't think it is appropriate to include so much itinerary data for demonstrating transitive in stories. Also, they shouldn't be included in the `__tests__` folder.